index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>LAMBORGHINI LANDING PAGE</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<header id="header">
+    <img 
+        src="https://upload.wikimedia.org/wikipedia/commons/thumb/1/18/Lamborghini_Sian_at_IAA_2019_IMG_0332.jpg/280px-Lamborghini_Sian_at_IAA_2019_IMG_0332.jpg" 
+        alt="" 
+        id="header-img"
+        width="100";
+        height= "100";
+        />
+    <nav id="nav-bar">
+        <a href="#photos" class="nav-link">Photos</a>
+        <a href="#about-us" class="nav-link">About us</a>
+        <a href="#pricing" class="nav-link">Pricing</a>
+    </nav>
+</header>
+<body>
```

Comment:
I would suggest you encapsulate the header and all other contents of the "body" of the HTML document within the body tag in order to follow best practices.